apps/site/components/SkuSelector/SkuSelectorColor.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useMemo } from 'react'
+import { SkuSelector, SkuSelectorProps, SkuOption } from '@faststore/ui'
+
+import { getSkuSlug } from './skuVariants'
+
+const SkuSelectorColor = () => {
+  const product = {
+    name: 'Tech Keyboard',
+    isVariantOf: {
+      name: 'Tech Keyboard',
+      productGroupID: '99995945',
+      skuVariants: {
+        activeVariations: {
+          Color: 'Blue',
+        },
+        slugsMap: {
+          'Color-Blue': 'tech-keyboard-99988216',
+          'Color-Yellow': 'tech-keyboard-99988212',
+          'Color-Gray': 'tech-keyboard-99988210',
+        },
+        availableVariations: {
+          Color: [
+            {
+              hexColor: '#6bb5ff',
+              alt: 'skuvariation',
+              label: 'Color: Blue',
+              value: 'Blue',
+            },
+            {
+              hexColor: '#e5c45f',
+              alt: 'skuvariation',
+              label: 'Color: Yellow',
+              value: 'Yellow',
+            },
+            {
+              hexColor: '#dfdfdf',
+              alt: 'skuvariation',
+              label: 'Color: Gray',
+              value: 'Gray',
+              disabled: true,
+            },
+          ],
+        },
+      },
+    },
+  }
+
+  const [skuPropertyName] = useMemo(() => {
+    return Object.keys(product.isVariantOf.skuVariants.activeVariations)
+  }, [product.isVariantOf.skuVariants.activeVariations])
+
+  const mountItemHref = useCallback(
```

Comment:
How about `getItemHref`? The word `mount` already has an associated meaning in the context of React components ("mounting a component")